src/app/views/apps/modals/app-connect-modal-myob.html

```diff
@@ -0,0 +1,29 @@
+<div id="app-connect-modal">
+  <div class="modal-header">
+    <div class="close" type="button" aria-label="Close" ng-click="close()">×</div>
+    <h3 class="modal-title">Connect to {{app.name}}</h3>
+  </div>
+  <div class="modal-body">
+    <h4 class="align-center">{{ "mno_enterprise.webhook.o_auth.providers.myob.which_version_used" | translate }}</h4>
+    <div class="spacer1"></div>
```

Comment:
We do not use `spacer*` classes anymore. Use `.top-buffer-*`. Please remove them everywhere.